orchagent/fdborch.h

```diff
@@ -14,19 +14,35 @@ struct FdbEntry
     {
         return tie(mac, bv_id) < tie(other.mac, other.bv_id);
     }
+    bool operator==(const FdbEntry& other) const
+    {
+        return tie(mac, bv_id) == tie(other.mac, other.bv_id);
+    }
 };
 
 struct FdbUpdate
 {
     FdbEntry entry;
     Port port;
+    string type;
```

Comment:
I am not sure following suggestion is perfect: If add two fields 'string type' and 'string port_name' in struct FdbEntry, it seems more simple, no need to add 'struct FdbData', no need to modify 'struct FdbUpdate' , 'struct SavedFdbEntry' and 'set<FdbEntry> m_entries'.

Reply:
Struct FdbEntry has only the key (mac and bv_id) and is used to lookup fdb in the tree as follows as an example:
auto it = m_entries.find(entry);
So, any other information other than key is stored separately.

Reply:
Operator '<' is overloaded in struct FdbEntry, '=' may also be overloaded.

Reply:
Thanks for the suggestion, for now its not needed.